tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -926,18 +926,6 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainHelpers.VolumeCornerIterator",
   test_vci_3d();
 }
 
-// [[OutputRegex, Index out of range.]]
-[[noreturn]] SPECTRE_TEST_CASE(
-    "Unit.Domain.DomainHelpers.VolumeCornerIterator.Assert", "[Domain][Unit]") {
```

Comment:
This change seems unrelated to the rest of the PR

Reply:
The test doesn't test anything new and actually tests an assertion in collapsed index instead of in the iterator itself. I need to do something with it, so I'm removing it.